src/screens/Topics/index.js

```diff
@@ -302,4 +304,4 @@ const styles = StyleSheet.create({
     textDecorationLine: 'underline'
   }
 });
-export default React.memo(Topics);
+export default Topics;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `React.memo` could potentially impact performance if this component re-renders often. If `Topics` is a heavy component or if its props change frequently, consider wrapping it with `React.memo` to avoid unnecessary re-renders.
```diff
- export default Topics;
+ export default React.memo(Topics);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->